cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,106 @@
+    <label class="label setting-label" for="list-license-types">
+        <%= gettext("License Type") %>
+    </label>
+    <ul class="license-types" id="list-license-types">
+        <% var link_start_tpl = '<a href="{url}" target="_blank">'; %>
+        <% _.each(licenseInfo, function(license, licenseType) { %>
+            <li data-license="<%- licenseType %>">
+                <button name="license-<%- licenseType %>"
+                        class="action license-button <% if(model.type === licenseType) { print("is-selected"); } %>"
+                        <% if (license.tooltip) { %>data-tooltip="<%- license.tooltip %>"<% } %>>
+                    <%- license.name %>
+                </button>
+                <p class="tip">
+                    <% if(license.url) { %>
+                        <a href="<%- license.url %>" target="_blank">
+                            <%= gettext("Learn more about {license_name}")
+                                        .replace("{license_name}", license.name)
+                            %>
+                        </a>
+                    <% } else { %>
+                        &nbsp;
+                    <% } %>
+                </p>
+            </li>
+        <% }) %>
+    </ul>
+
+<% var license = licenseInfo[model.type]; %>
+<% if(license && !_.isEmpty(license.options)) { %>
+    <div class="wrapper-license-options">
+        <label class="label setting-label" for="list-license-options">
+            <%- gettext("Options for {license_name}").replace("{license_name}", license.name) %>
+        </label>
+        <p class='tip tip-inline'>
+            <%- gettext("The following options are available for the {license_name} license.")
+                        .replace("{license_name}", license.name) %>
+        </p>
+        <ul class="license-options" id="list-license-options">
+            <% _.each(license.option_order, function(optionKey) { %>
+                <% var optionInfo = license.options[optionKey]; %>
+                <% if (optionInfo.type == "boolean") { %>
+                    <% var optionSelected = model.options[optionKey]; %>
+                    <% var optionDisabled = optionInfo.disabled ||
+                            (optionInfo.conflictsWith && _.any(optionInfo.conflictsWith, function(key) {return model.options[key];}));
+                    %>
+                    <li data-option="<%- optionKey %>"
+                            class="action-item license-button
+                                   <% if (optionSelected) { print("is-selected"); } %>
+                                   <% if (optionDisabled) { print("is-disabled"); } else { print("is-clickable"); } %>"
+                    >
+                        <i class="fa fa-fw
+                             <% if(optionSelected) { print("fa-check-square-o"); } else { print("fa-square-o"); } %>
+                             <% if(optionDisabled) { print("is-disabled"); } %>
+                             "></i>
```

Comment:
are these states (disabled or selected) indicated in any other way than the icon?  If not, we should add accessible text.  Since the icon is sufficient for conveying the state to sighted users, the state could be indicated with offscreen text AKA screen reader text ```<span class="sr">screen reader text</span>```